sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java

```diff
@@ -111,6 +108,9 @@ public void buttonClick(final ClickEvent event) {
 		menuItemsLayout.addStyleName("valo-menuitems");
 		menuArea.addComponent(menuItemsLayout);
 
+		if (contentArea instanceof VerticalLayout) {
+			((VerticalLayout) contentArea).setMargin(false);
```

Comment:
I think I already made this change in the master branch. 

https://github.com/tsuoanttila/sidemenu-addon/blob/master/sidemenu-addon/src/main/java/org/vaadin/teemusa/sidemenu/SideMenu.java#L161